knowledge/technical_manual/Ceph/upstream/RBD/snapshot/qna.yaml

```diff
@@ -0,0 +1,98 @@
+version: 3
+created_by: Sunil Angadi
+domain: opensource_storage
+seed_examples:
```

Comment:
Hi Sunil,
you might need to add this 
`document_outline: Teach the Large Language Model about the Deploying MDS of CephFS
document:
  repo: git@github.com:pavankumarag/ceph-instructlab-taxonomy-data.git
  commit: b60d0d0
  patterns: [upstream/doc/cephfs/add-remove-mds*.md]`
  
  repo and commit is common for every one , patterns should point to your md file